src/convexhull.jl

```diff
@@ -0,0 +1,30 @@
+"""
+Compute the upper part of the convex hull of a given set of 2D points 'pset' using the "Graham Scan" algorithm.
+
+*) Sort the points lexicographic
+
+*) Assess the last 3 points in 'upperHull'. If the the clockwise angle is <180° remove the second to last point.
+
+"""
+
+function upperhull(pset::PointSet) 
+
+  xs=sort(coordinates.(pset), by=first)
+
```

Comment:
Please read the collaboration guidelines, specifically the code style of the project: https://github.com/JuliaGeometry/Meshes.jl#contributing

Reply:
Sorry, I don't know what I should change here exactly

Reply:
I will give it a another round of reviews so that we can align the code style further.